lib/widgets/organization_list.dart

```diff
@@ -69,8 +69,9 @@ class OrganizationList extends StatelessWidget {
               print(model.organizations.isEmpty);
               if (model.organizations.isEmpty) {
                 navigationServiceLocal.showTalawaErrorDialog(
-                    "No organizations found ! Please contact your admin ",
-                    MessageType.error);
+                  "No organizations found ! Please contact your admin ",
```

Comment:
Add multilingual responses.

Reply:
@palisadoes I actually scheduled to do this for today. I was busy with college assignments for the past 2 days.
 But now It's merged, so  I'll add the multilingual responses in my future PR 👍🏻

Reply:
There is a new issue created to fix this